hbase-zookeeper/pom.xml

```diff
@@ -274,6 +284,16 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-common</artifactId>
+          <exclusions>
```

Comment:
Can we add these exclusions in the parent pom? No?

Reply:
Unfortunately, that doesn't work

Reply:
Oh you mean just exclude from parent pom and we don't need to exclude from individual modules right? Sure let me do that for jackson-jaxrs and jackson-xc since they are common across all modules

Reply:
Yes, that's what I mean, excluding in the parent pom so the sub modules just inherit the exclusions automatically. But I'm not a maven expert so please confirm whether it actually works...

Reply:
@Apache9 this doesn't work as expected. For instance, if downstream app has hbase-client dependency and if we have just excluded from parent pom, downstream would still get vulnerable Jackson1 dependencies from hbase-client. In fact, the same is done for branch-1: https://github.com/apache/hbase/commit/4b34d24f7a12510f69cf4d2e190359dc0b271ead

Reply:
OK for now but this is a bit strange. At least the downstreamer need to find the version from parent pom, then why it just ignores the exclusions?

Reply:
And what I mean is to exclude them in the dependencyManagement section in the parent pom, not in dependency section(and I wonder whether we have this section in the parent pom...)

Could you please try again?

Reply:
You are correct @Apache9 
It works...Thanks. Just committed the changes

Reply:
I have committed all the necessary and suggested changes.